kubediff

```diff
@@ -65,7 +65,8 @@ class ParseArgs():
                             dest='exit_on_diff',
                             default=True)
 
-        parser.add_argument('path', nargs='*')
+        parser.add_argument('path', nargs='*', help='path(s) from which '
```

Comment:
Shall we rename the variable to 'paths' as well?

Reply:
changed 🥇